SPECS-EXTENDED/openblas/openblas.spec

```diff
@@ -57,8 +61,25 @@ BuildRequires:  multilib-rpm-config
 BuildRequires:  /usr/bin/execstack
 %endif
 
+# LAPACK
+%if %{with system_lapack}
+%if 0%{?rhel} == 5 || 0%{?rhel} == 6
 BuildRequires:  lapack-devel
+%else
+BuildRequires:  lapack-static
+%endif
+# Do we have LAPACKE? (Needs at least lapack 3.4.0)
+%if 0%{?fedora}
```

Comment:
Can we remove some of the checks for non-Mariner distros in this spec, and tailor this more to our needs? I know we keep a lower standard for our extended specs, but removing the conditionals goes a long way for readability, especially for longer and more complicated specs like this one.